keyboardvisibilityevent/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion "24.0.1"
 
     defaultConfig {
-        minSdkVersion 9
+        minSdkVersion 16
```

Comment:
we should use `removeGlobalOnLayoutListener` for older version. 

and `Application.ActivityLifecycleCallbacks` requires API 14 :(
...But maybe it's time to forget about pre-ICS ;)